### PR TITLE
Fix log levels and statements

### PR DESCRIPTION
**Pull request by nielsm5**

### CODE REVIEW

File: core/src/main/java/org/frankframework/util/RunStateManager.java [hunk 1]
```diff
@@ -34,8 +34,7 @@ public synchronized RunState getRunState() {
 
 	public synchronized void setRunState(RunState newRunState) {
 		if(runState != newRunState) {
-			if(log.isDebugEnabled())
-				log.debug("Runstate [" + this + "] set from " + runState + " to " + newRunState);
+			log.debug("Runstate [{}] set from {} to {}", super.toString(), runState, newRunState);
```
Comment (nielsm5) [inline core/src/main/java/org/frankframework/util/RunStateManager.java:37]: Fixes double current state (inherited from `toString`):
`util.RunStateManager - Runstate [RunStateManager@75a9bbef [STOPPED]] set from STOPPED to STARTING`

File: core/src/main/java/org/frankframework/management/bus/MessageDispatcher.java [hunk 2]
```diff
@@ -116,7 +117,7 @@ private void findServiceActivators(BeanDefinition beanDef) throws ClassNotFoundE
 	}
 
 	private void registerServiceActivator(Object bean, Method method, SubscribableChannel channel, BusTopic topic) {
-		String componentName = bean.getClass().getSimpleName()+"."+method.getName();
+		String componentName = ClassUtils.classNameOf(bean)+"."+method.getName();
```
Comment (nielsm5) [inline core/src/main/java/org/frankframework/management/bus/MessageDispatcher.java:120]: Fixes `... ServiceActivator [GetSchedules$$EnhancerBySpringCGLIB$$deff747.getSchedules] on ...`
By removing the EnhancerBySpringCGLIB

File: core/src/main/java/org/frankframework/lifecycle/VerifyDatabaseConnectionBean.java [hunk 3]
```diff
@@ -83,9 +86,12 @@ public void afterPropertiesSet() throws Exception {
 
 	private @Nonnull PlatformTransactionManager getTransactionManager() {
 		try {
-			PlatformTransactionManager txManager;
-			txManager = applicationContext.getBean("txManager", PlatformTransactionManager.class);
-			log.info("found transaction manager to [{}]", txManager);
+			final PlatformTransactionManager txManager = applicationContext.getBean("txManager", PlatformTransactionManager.class);
+
+			if(log.isInfoEnabled()) {
```
Comment (nielsm5) [inline core/src/main/java/org/frankframework/lifecycle/VerifyDatabaseConnectionBean.java:91]: Log the actual tx manager

File: core/src/main/java/org/frankframework/jdbc/datasource/TransactionalDbmsSupportAwareDataSourceProxy.java [hunk 4]
```diff
@@ -44,8 +44,8 @@ public TransactionalDbmsSupportAwareDataSourceProxy(DataSource delegate) {
 
 	public Map<String, String> getMetaData() throws SQLException {
 		if (metadata == null) {
-			log.debug("populating metadata from getMetaData");
-			try (Connection connection = getConnection()) {
+			try (Connection connection = super.getConnection()) {
+				log.trace("populating metadata from getMetaData");
```
Comment (nielsm5) [inline core/src/main/java/org/frankframework/jdbc/datasource/TransactionalDbmsSupportAwareDataSourceProxy.java:48]: Reduce spam, this ain't actually that interesting and is solely here for debugging purposes...

**Review by tnleeuw — APPROVED** [review]

**Review by jkosternl — APPROVED** [review]

**Pull request merged**